var/spack/repos/builtin/packages/intel-oneapi-compilers/package.py

```diff
@@ -5,40 +5,39 @@
 
 import glob
 import subprocess
+from os import path
+from sys import platform
 
 from spack import *
 
 
-releases = {'2021.1.0':
-            {'irc_id': '17427', 'build': '2684'}}
-
-
 class IntelOneapiCompilers(IntelOneApiPackage):
-    """Intel oneAPI compilers.
-
-    Contains icc, icpc, icx, icpx, dpcpp, ifort, ifx.
-
-    """
-
-    maintainers = ['rscohn2']
+    """Intel OneAPI compilers
 
-    homepage = 'https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/dpc-compiler.html'
+    Provides Classic and Beta compilers for: Fortran, C, C++"""
 
-    version('2021.1.0', sha256='666b1002de3eab4b6f3770c42bcf708743ac74efeba4c05b0834095ef27a11b9', expand=False)
+    homepage = "https://software.intel.com/content/www/us/en/develop/tools/oneapi.html"
 
     depends_on('patchelf', type='build')
 
+    if platform == 'linux':
+        version('2021.1.2',
+                sha256='68d6cb638091990e578e358131c859f3bbbbfbf975c581fd0b4b4d36476d6f0a',
+                url='https://registrationcenter-download.intel.com/akdlm/irc_nas/17513/l_dpcpp-cpp-compiler_p_2021.1.2.63_offline.sh',
+                expand=False)
+        resource(name='fortran-installer',
+                 url='https://registrationcenter-download.intel.com/akdlm/irc_nas/17508/l_fortran-compiler_p_2021.1.2.62_offline.sh',
+                 sha256='29345145268d08a59fa7eb6e58c7522768466dd98f6d9754540d1a0803596829',
+                 expand=False,
+                 placement='fortran-installer',
+                 when='@2021.1.2')
+
     def __init__(self, spec):
-        self.component_info(
-            dir_name='compiler',
-            components=('intel.oneapi.lin.dpcpp-cpp-compiler-pro'
-                        ':intel.oneapi.lin.ifort-compiler'),
-            releases=releases,
-            url_name='HPCKit')
+        self.component_info(dir_name='compiler')
         super(IntelOneapiCompilers, self).__init__(spec)
 
-    def _join_prefix(self, path):
-        return join_path(self.prefix, 'compiler', 'latest', 'linux', path)
+    def _join_prefix(self, p):
+        return path.join(self.prefix, 'compiler/latest/linux', p)
```

Comment:
(minor) The original call

```
join_path(self.prefix, 'compiler', 'latest', 'linux', path)
```

ensures that the proper file separators are used. As I understand it this is only ever likely to run on systems that use `/` as a file separator, so this isn't a big deal (but it is idiomatic to avoid using separators in calls to `os.path.join`).

Reply:
eliminated all / in the file